src/ui/public/react_components.js

```diff
@@ -0,0 +1,9 @@
+import 'ngreact';
+
+import { PlusIcon } from 'ui_framework/components/icon/plus_icon';
+import { TrashIcon } from 'ui_framework/components/icon/trash_icon';
```

Comment:
Can we implement an `index.js` file in `ui_framework/components` which imports and re-exports the components?

```javascript
export { PlusIcon } from './icon/plus_icon';
export { TrashIcon } from './icon/trash_icon';
```

And then we can import them here, like this:

```javascript
import {
  PlusIcon,
  TrashIcon,
} from 'ui_framework/components';
```

I think this is an improvement because then we can treat the UI Framework like a module, and we'll have the ability to rearrange things internally by renaming folders and moving files without changing the interface. We can also write tests for this interface for immediate notification of when we break it by accident.